internal/distro/rhel8/pipelines.go

```diff
@@ -935,7 +935,13 @@ func simplifiedInstallerTreePipeline(repos []rpmmd.RepoConfig, packages []rpmmd.
 		dracutStageOptions.Install = []string{"/fdo_diun_pub_key_root_certs.pem"}
 	}
 	p.AddStage(osbuild.NewDracutStage(dracutStageOptions))
-
+	greenbootConfig := &osbuild.GreenbootConfig{
+		MonitorServices: []string{"sshd", "NetworkManager"}, //sshd,NetworkManager are default services to be monitored by greenboot
+	}
+	if len(greenboot.MonitorServices) != 0 {
+		greenbootConfig = new(osbuild.GreenbootConfig)
```

Comment:
not sure what this is doing, you need to actually create a config with the passed in services

Reply:
corrected and moved to osPipeLine